tools/cli/commands/connect.py

```diff
@@ -277,6 +277,13 @@ def connect_and_check():
             return
         if remaining_reconnects == 0:
             return
+        # Before we try to reconnect, check to see if the VM is still running.
+        status, metadata_items = utils.describe_instance(
```

Comment:
I would rename `metadata_items` to `unused_metadata_items` to make it clear that we don't need that part of the return value.